src/src/services/keycloak/keycloak.service.ts

```diff
@@ -0,0 +1,74 @@
+import { HttpService } from '@nestjs/axios';
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { AxiosRequestConfig } from 'axios';
+
+@Injectable()
+export class KeycloakService {
+
+    constructor(private configService: ConfigService, private readonly httpService: HttpService) { }
+
+    public async getAdminKeycloakToken() {
+        console.log("inside getAdminKeycloakToken")
+        const data = {
+            username: 'admin',
+            client_id: 'admin-cli',
+            grant_type: 'client_credentials',
+            password: this.configService.get<string>('KEYCLOAK_ADMIN_PASSWORD'),
```

Comment:
for `grant_type: 'client_credentials',` you dont need to send password

Reply:
done